src/content/docs/extensions/json.mdx

```diff
@@ -548,34 +548,44 @@ Output:
 ```
 
 ```cypher
-RETURN to_json({duck : 42});
+RETURN to_json({kuzu : 42});
```

Comment:
Could we give a more realistic example using one of the example JSON fields from above? Say Person name and age.

Reply:
changed to real examples